services/QuillLMS/engines/evidence/spec/services/evidence/gen_ai_record_builder_spec.rb

```diff
@@ -33,11 +33,13 @@ module Evidence
         expect(gen_ai_activity.but_text).to eq(relevant_texts[:but_text])
       end
 
-      it 'should create a stem vault for each prompt' do
+      it 'should create a stem vault and relevant text for each prompt' do
         prompts.each do |prompt|
           stem_vault = Evidence::Research::GenAI::StemVault.find_by(prompt_id: prompt.id)
           expect(stem_vault).not_to be_nil
           expect(stem_vault.stem).to eq(prompt.text.rpartition(prompt.conjunction).first.strip)
+          relevant_text = Evidence::Research::GenAI::RelevantText.find_by(text: stem_vault.relevant_text, stem_vault:)
```

Comment:
You could combine these into one line
`expect(Evidence::Research::GenAI::RelevantText.exists?(text: stem_vault.relevant_text, stem_vault:)).to be_true`